test/gtest/ucs/test_async.cc

```diff
@@ -810,6 +819,11 @@ UCS_TEST_P(test_async_timer_mt, multithread) {
     EXPECT_GE(min_count, exp_min_count);
 }
 
+std::ostream& operator<<(std::ostream& os, ucs_async_mode_t mode)
```

Comment:
where do you use it?

Reply:
it's used (implicitly) by INSTANTIATE_xx to stringify the mode parameter and print it while test is running
without this it will be printed as number